js/test/strings-ext/testlistfmt_sv.js

```diff
@@ -108,7 +108,7 @@ module.exports.testlistfmt_bs = {
     
```

Comment:
the wrapper should be `module.exports.testlistfmt_sv` instead of `module.exports.testlistfmt_bs`

Reply:
oh yeah, it should. I update it.